### ISSUE TITLE
Exception not logged properly (causing NPE while logging)

### ISSUE THREAD

**Issue by PiotrCL** [nguyenq/tess4j#41]
The catch block in PdfUtilities.convertPdf2Png() is trying to log the Exception 

https://github.com/nguyenq/tess4j/blob/master/src/main/java/net/sourceforge/tess4j/util/PdfUtilities.java#L107
`
logger.error(e.getCause().toString(), e);
`

e.getCause() may be null, calling toString on null is throwing NPE

Please check:
http://stackoverflow.com/questions/39504263/tesseract-error-net-sourceforge-tess4j-tesseract-null/39567186#39567186

**Comment by 4F2E4A2E** [comment]
Hello Guys! Thank you very mch for the Stackoverflow thread. We will fix this and maybe by the end of the week we have a snapshot release.

**Comment by nguyenq** [comment]
Fix in tess4j-3.2.2 release.

**Comment by kernelfreak** [comment]
Hello, This is not fixed, I get the same error in 3.4. I tried new Tesseract as well as Tesseract.getInstance();

Error log, 

```
net.sourceforge.tess4j.TesseractException: java.lang.IndexOutOfBoundsException
	at net.sourceforge.tess4j.Tesseract.doOCR(Tesseract.java:215)
	at net.sourceforge.tess4j.Tesseract.doOCR(Tesseract.java:196)
	at com.tooltank.spring.service.GroupAttachmentsServiceImpl.testOcr(GroupAttachmentsServiceImpl.java:909)
	at com.app.spring.service.GroupAttachmentsServiceImpl.lambda$addAttachment$1(GroupAttachmentsServiceImpl.java:505)
	at java.lang.Thread.run(Thread.java:745)
```

I would recommend re-opening this issue.

**Comment by nguyenq** [comment]
@kernelfreak, are you sure it's the same error? It looks different to me.
